owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModelTest.kt

```diff
@@ -163,4 +163,27 @@ class SettingsSecurityViewModelTest : ViewModelTest() {
         }
     }
 
+    @Test
+    fun `is lock delay enforced enabled - ok - true`() {
+        every { contextProvider.getInt(any()) } returns 1
+
+        val isEnabled = securityViewModel.isLockDelayEnforcedEnabled()
+        assertTrue(isEnabled)
+
+        verify(exactly = 1) {
+            contextProvider.getInt(R.integer.lock_delay_enforced)
+        }
+    }
+
+    @Test
+    fun `is lock delay enforced enabled - ok - false`() {
```

Comment:
Thanks for changing the name of these 2 tests, but could you change as well the one of the 2 previous tests which are wrongly named too? 👍

Reply:
Oh! I forgot them! 😞 , thank you! renamed!